scripts/ci/changelog/templates/template.md.tera

```diff
@@ -5,7 +5,7 @@
 {% if env.PRE_RELEASE == "true" -%}
 This pre-release contains the changes from `{{ env.REF1 | replace(from="refs/tags/", to="") }}` to `{{ env.REF2 | replace(from="refs/tags/", to="") }}`.
 {%- else -%}
-This release contains the changes from `{{ env.REF1 }}` to `{{ env.REF2 }}`.
+This release contains the changes from `{{ env.REF1 | replace(from="refs/tags/", to="") }}` to `{{ env.REF2 | replace(from="refs/tags/", to="") }}`.
```

Comment:
I did forget that case and added it in https://github.com/paritytech/polkadot/pull/5501/files but I take it either way :)